packages/mui-joy/src/Checkbox/Checkbox.tsx

```diff
@@ -322,6 +322,7 @@ const Checkbox = React.forwardRef(function Checkbox(inProps, ref) {
       name,
       value,
       readOnly,
+      role: 'checkbox',
```

Comment:
It's redundant to have `role="checkbox"` on an `input type="checkbox"`

```suggestion
      role: undefined,
```